dags/burnham.py

```diff
@@ -178,14 +242,35 @@
   AND metrics.uuid.test_run = "{test_run}"
 """
 
+STARBASE46_SENSOR_TEMPLATE = """
+SELECT
+  COUNT(*) >= 1
+FROM
+  `{project_id}.burnham_live.starbase46_v1`
+WHERE
+  submission_timestamp > TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 3 HOUR)
+  AND metrics.uuid.test_run = "{test_run}"
+"""
+
+SPACE_SHIP_READY_SENSOR_TEMPLATE = """
+SELECT
+  COUNT(*) >= 3
+FROM
+  `{project_id}.burnham_live.space_ship_ready_v1`
+WHERE
+  submission_timestamp > TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 3 HOUR)
+  AND metrics.uuid.test_run = "{test_run}"
```

Comment:
We use a parameter in the burnham-bigquery operators because we have to encode the SQL queries before Airflow generates the test run UUID and renders the according template. It's not ideal. 

Can we use parameters for Airflow sensor operators? I can't seem to find anything in the Airflow documentation.

https://airflow.apache.org/docs/stable/_api/airflow/sensors/base_sensor_operator/index.html

Reply:
This is a custom operator [we made](https://github.com/mozilla/telemetry-airflow/blob/master/dags/operators/bq_sensor.py), but it would take a but of finagling to get parameters to work. Let's leave as-is.